content/docs/developer/deploy-nodejs/_index.md

```diff
@@ -1,137 +1,154 @@
+
+- An [account](/account-holders//create-acct/) that has been [funded](/account-holders/fund-acct/) with auton, to pay for transaction gas costs. You will need the  private key of the account to unlock the account in the JavaScript environment.
+
+- Configuration details for the Autonity network you are deploying to: a [public Autonity network](/networks/) or a [custom network](/developer/custom-networks/) if you are deploying to a local testnet.
 
-## Setup
+## Setup your working environment
 
-Create a new directory and initialise the project with npm:
+1. Create a working directory and initialise the project with `npm`:
 
 ```bash
 mkdir ERC20token && cd ERC20token
 npm init -y
 ```
 
-Next install OpenZeppelin Contracts, and other required modules. OpenZeppelin is an open source library of smart contracts.
+2. Install OpenZeppelin Contracts and `web3` modules:
 
 ```bash
 npm i --save-dev @openzeppelin/contracts
 npm i web3
 ```
 
+The `web3` module is used to add a private key. OpenZeppelin is an open source library of smart contracts.
+
 ## Write the deploy script
 
-Add the following script `deploy.js`, filling in the RPC node url, and the private key of the account you are using to make the transaction. The arguments used to deploy the ERC20 token are set within the `deployTokens` function.
-
-```javascript
-const Web3 = require("web3");
-const ERC20 = require("./node_modules/@openzeppelin/contracts/build/contracts/ERC20PresetFixedSupply.json");
-
-const RPC = "<NODE URL>";
-const prvKey = "<PRIVATE KEY>";
-const GasLimit = 8000000;
-
-// deploy ERC20 contract
-async function deployTokens(web3, sender) {
-    try {
-      let token = new web3.eth.Contract(ERC20.abi);
-
-      token = await token
-        .deploy({
-          data: ERC20.bytecode,
-          arguments: [
-            "mytoken",
-            "TKN",
-            web3.utils.toWei("9999999999999999999", "ether"),
-            sender,
-          ],
-        })
-        .send({ from: sender, gas: GasLimit })
-        .on("receipt", function (receipt) {
-            console.log("receipt", receipt);
-          });
-
-      console.log("token address: ", token.options.address);
-
-      return [token.options.address];
+3. Add the following script `deploy.js` to your working directory, where:
+	- `<NODE_URL>` is the rpc endpoint of the node you are connecting to.
+	- `<PRIVATE_KEY>` is the private key of the account you are using to submit the transaction.
+	- `<TOKEN_NAME>` is the text label name for your token.
+	- `<TOKEN_SYMBOL>` is the text symbol acronym for your token.
+	
+	{{< alert title="Warning" color="warning" >}}
+	Including the private key as clear text is done in this example only because this is a testnet setting without real value. Putting a private key in the clear text is **not** recommended.
```

Comment:
I think that the indentation causes it to show as a code block
```suggestion
Including the private key as clear text is done in this example only because this is a testnet setting without real value. Putting a private key in the clear text is **not** recommended.
```

Reply:
Yes!

Reply:
Do we really need this private key here?  
If we use he snippet here: https://web3py.readthedocs.io/en/stable/web3.eth.account.html#extract-private-key-from-geth-keyfile  we can decrypt a keyfile.  Although I'd expect truffle should provide a nicer approach to this.